Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -83,6 +83,11 @@
   public static final String MIN_SDK_VERSION = "1.12";
   private static final String MIN_MOVE_FILE_SDK_VERSION = "2.3.2";
 
+  // Webdev works going back to 2.6.0, future minimum version listed in the pubspec.yaml, link below, won't mean that 2.6.0 aren't
+  // supported.
+  // https://github.com/dart-lang/webdev/blob/master/webdev/pubspec.yaml#L11
+  private static final String MIN_WEBDEV_SDK_VERSION = "2.6.0";
```

Comment:
I noticed that we already have `DartWebdevParameters.MIN_DART_SDK_VERSION_FOR_WEBDEV`, though it's set to "2.5".  Will 2.5 be enough for Webdev run configuration? Let's merge 2 constants into 1 if so.